frontends/election-manager/src/app_root.tsx

```diff
@@ -294,15 +292,11 @@ export function AppRoot({
   }
 
   const saveTranscribedValue = useCallback(
-    async (
-      ballotId: BallotId,
-      contestId: ContestId,
-      transcribedValue: string
-    ) => {
+    async (adjudicationId: string, transcribedValue: string) => {
       try {
-        await fetch(`/admin/write-ins/transcribe`, {
-          method: 'POST',
-          body: JSON.stringify({ ballotId, contestId, transcribedValue }),
+        await fetch('/admin/write-ins/adjudication/transcribe', {
```

Comment:
REST-wise, this would be more like `/admin/write-ins/adjudications/${adjudicationId}/transcription`.